tests/61push/01message-pushed.pl

```diff
@@ -630,3 +630,103 @@ sub check_received_push_with_name
          );
       });
    };
+
+test "Invites over federation are correctly pushed",
```

Comment:
Are we making more Sytest tests because there isn't a good way to test push gateway stuff in Complement?